tests/phpunit/Compat/WC_Stripe_Subscriptions_Helper_Test.php

```
@@ -335,4 +359,72 @@ public function test_get_detached_payment_data_from_subscription() {
+	public function test_is_subscription_edit_page( bool $hpos_enabled, array $query_params, bool $expected ): void {
+		update_option( 'woocommerce_custom_orders_table_enabled', $hpos_enabled ? 'yes' : 'no' );
+
+		$_REQUEST = $query_params;
+
+		$actual = WC_Stripe_Subscriptions_Helper::is_subscription_edit_page();
+
+		// Clean up.
+		unset( $_REQUEST );
+
+		$this->assertSame( $expected, $actual );
+	}
```

PHPMD (2.15.0)

[error] test_is_subscription_edit_page accesses the super-global variable $_REQUEST. (undefined)

(Superglobals)